schema/home/fetch.js

```diff
@@ -22,6 +23,12 @@ export const featuredFair = () => {
   });
 };
 
+export const activeSaleArtworks = (accessToken) => {
+  return gravity.with(accessToken)('me/lot_standings').then((results) => {
```

Comment:
We may want to pass `live=true` into this endpoint, to grab only the sale artworks with bids in ongoing sales: https://github.com/artsy/gravity/blob/master/app/api/v1/me_lot_standings_endpoint.rb#L16